lib/components/Loader/Spinner.module.scss

```diff
@@ -51,15 +51,14 @@
         width: 100%;
         height: 100%;
         opacity: 1;
-        // transform: rotate(45deg);
     }
     
     #{$inner-class-name} > div {
         @include md-box(inline-block, relative);
         width: $diameter;
         height: $diameter;
         border-radius: $diameter;
-        background-color: $color;
+        background-color: var(--color-text-rest);

```

Comment:
Before the `background-color` would be set to black, now because you're setting it to `color-text-rest` it'll be set to `$color-white`... is this ok?

Reply:
It will be black in light theme, but it will be light in dark theme... This seemed to be a bug with this control, so as part of this PR I also fixed it to work fine in all themes